openmc/universe.py

```diff
@@ -87,7 +87,8 @@ def add_volume_information(self, volume_calc):
                 self._volume = volume_calc.volumes[self.id].n
                 self._atoms = volume_calc.atoms[self.id]
             else:
-                raise ValueError('No volume information found for this universe.')
+                raise ValueError(
+                    'No volume information found for this universe.')
```

Comment:
I personally prescribe to the [90-ish character](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length) line length philosophy. To me, this line looks perfectly fine as it is and wrapping it onto a second line doesn't make it any easier to read. Certainly there's a point where that ceases to be true.

Reply:
I'm always keen to add black as a github action to automate this sort of decision making or flake8 if that is preferred.

Reply:
I'm just going to run autopep8 and call it a day. If we want to make changes to the PEP8 policy, let's put it in the docs and then I'll change my config files for autopep8.